web/webapp/src/main/java/no/nav/foreldrepenger/melding/web/app/tjenester/ForvaltningRestTjeneste.java

```diff
@@ -75,20 +76,19 @@ public Response testeDokgen() {
         FellesDokumentdata fellesDokumentdata = new FellesDokumentdata.Builder()
```

Comment:
For min del må du gjerne slette denne tjenesten nå, så slipper vi å vedlikeholde den

Reply:
ok